angular2-jwt.ts

```diff
@@ -304,56 +304,34 @@ export function tokenNotExpired(tokenName = AuthConfigConsts.DEFAULT_TOKEN_NAME,
   return token != null && !jwtHelper.isTokenExpired(token);
 }
 
-export const AUTH_PROVIDERS: Provider[] = [
```

Comment:
Even though `AUTH_PROVIDERS` isn't compatible with AoT, does it hurt to have it in the lib until we make a big breaking release? Just because it's there doesn't mean that people need to use it. But for those who are still using it and who aren't pinned to a patch version in their dependencies, taking it out will break them.